config.ru

```diff
@@ -0,0 +1,14 @@
+require 'puppet_webhook'
+require 'logger'
+
+LOGGER = Logger.new('/var/log/puppet_webhook').freeze
+COMMAND_PREFIX = 'umask 0022;'.freeze
+LOCKFILE = '/var/run/puppet_webhook/puppet_webhook.lock'.freeze
+
+FileUtils.makedirs('/var/run/puppet_webhook') unless File.exist?('/var/run/puppet_webhook')
```

Comment:
The conditional shouldn't be necessary. The `mkdir -p` operation only does something if the directory doesn't exist. This actually also reduces reliability, since `File.exist?` will return `true` whether the path is a directory or not.